pubsub/envelope.go

```diff
@@ -89,28 +89,26 @@ func NewCloudEventsEnvelope(id, source, eventType, subject string, topic string,
 		Topic:           topic,
 		PubsubName:      pubsubName,
 		Data:            j,
+		DaprTraceID:     traceID,
 	}
 }
 
-// IsCloudEvent returns a bool to indicate if a payload is a CloudEvents compliant format
-func IsCloudEvent(data []byte) bool {
-	var j interface{}
-	err := jsoniter.Unmarshal(data, &j)
+// SetTraceID adds the daprtraceid field to an existing cloudevents JSON
+func SetTraceID(data []byte, traceID string) ([]byte, error) {
```

Comment:
This method seems to be useful only for tests, please move it to a test specific place. I would like to avoid production code messing with the payload after it has been serialized to bytes.